src/QueryFilter/AuditLog/UserFilter.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of Packagist.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *     Nils Adermann <naderman@naderman.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\QueryFilter\AuditLog;
+
+use App\Entity\User;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\HttpFoundation\InputBag;
+
+class UserFilter extends AbstractAdminAwareTextFilter
+{
+    protected function applyFilter(QueryBuilder $qb, string $paramName, string $pattern, bool $useWildcard): QueryBuilder
+    {
+        $qb->setParameter($paramName, $pattern);
+
+        $qb->innerJoin(User::class, 'targetUser', 'WITH', 'a.userId = targetUser.id');
+
+        if ($useWildcard) {
+            $qb->andWhere('targetUser.username LIKE :' . $paramName);
+        } else {
+            $qb->andWhere('targetUser.username = :' . $paramName);
+        }
```

Comment:
Why not using `attributes->>$.user.username` here to filter? I think it'd be best because the user might not exist anymore in the users table.